### DISCUSSION TITLE
Creating a dataset without a resource

### DISCUSSION THREAD

**Discussion by raz-shay** [ckan/ckan#7114] [Q&A]
Hello,
I would like to create a dataset from the ckan ui but without having to supply a resource (I will add resource/s to this dataset using ckan API on later time), currently when I add a new dataset I also have to supply at least one resource in order to complete the process.
Is this behavior can be customize and how ?
Thx
 Raz

**Comment by mutantsan** [comment]
https://docs.ckan.org/en/2.9/maintaining/configuration.html#ckan-dataset-create-on-ui-requires-resources

**Comment by Alex-Pavlyuk** [comment]
@raz-shay the solution from the post above should be enough for you to solve your first issue, it's easy
